quipucords/scanner/network/inspect.py

```diff
@@ -78,16 +78,16 @@ def run(self, manager_interrupt):
 
         self.connect_scan_task = self.scan_task.prerequisites.first()
         if self.connect_scan_task.status != ScanTask.COMPLETED:
-            error_message = 'Prerequisites scan task %d failed.' %\
-                self.connect_scan_task.sequence_number
+            error_message = 'Prerequisites scan task %d failed.' % \
+                            self.connect_scan_task.sequence_number
```

Comment:
Now it makes sense why linters were looking to change some strings on this file to be reformatted.

I'm not against commits making changes to formatting, but I believe this kind of change should live in a separate commit. 
That said, don't need to change this particular commit.